index.js

```diff
@@ -259,7 +267,7 @@ function processJSXElement(annotateFragments, t, jsxElement, componentName, sour
   const children = jsxElement.get('children')
   if (children && children.length) {
     let shouldSetComponentName = annotateFragments
-    for (let i=0; i < children.length; i += 1){
+    for (let i = 0; i < children.length; i += 1) {
       // Children don't receive the data-component attribute so we pass null for componentName unless it's the first child of a Fragment with a node and `annotateFragments` is true
       if (shouldSetComponentName && children[i].get('openingElement') && children[i].get('openingElement').node) {
         shouldSetComponentName = false
```

Comment:
Actually, why do we only annotate the first child of the fragment? According to the [react doc for fragments](https://reactjs.org/docs/fragments.html) the main purpose is to include a _list_ of children. I feel like we should annotate all the children of they result in an html element, right?